eduvarjaor/talk-text
# Firebase configuration

## Description

Created Firebase configuration setup with the proper credentials. 

## **Related Issue:**

N/A

## **Main changes explained:**

- Export Vue.js content to the client folder
- Create a server folder with Firebase and Openai settings files

## **How to test:**

1. check into current branch
2. do `npm install` to run this PR locally
3. run `firebase serve --only functions`
4. test the "http://localhost:5000/talk-text-21c1b/us-central1/transcriptAudio" endpoint at Postman or any other proper tool

## **Screenshots or videos of changes:**

If you get the response "OK", everything is working as intended.

![Screenshot_1](https://github.com/eduvarjaor/talk-text/assets/86692035/1a6fd5dd-67ce-4869-817f-890101911e05)
